src/data.toml

```diff
@@ -1,18 +1,22 @@
+get_addons = "Bunny offers compatibility with Vendetta & Revenge plugins and themes:\n\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\nYou are free to request plugin urls to be added to this list\n\n- You may install any  [Vendetta](<https://discord.gg/rJgnCJ3Kzm>)/[Enmity](<https://discord.gg/enmity>)/[Revenge](<https://discord.com/invite/ddcQf3s2Uq>) themes you like, do note that old themes may be incompatible\n\nYou may find a curated list of broken plugins in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
+installation_instructions = "### Android\n\n- **Root** with LSPosed - [BunnyXposed](<https://github.com/pyoncord/BunnyXposed/releases/latest>)\n- **Non-root** - [BunnyManager](<https://github.com/pyoncord/BunnyManager/releases/latest>)\n### iOS\n\n- **Jailed**\n - Pre-patched `.ipa` - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n - AltSource Repo - [Pyoncord App Repo](<https://raw.githubusercontent.com/pyoncord/BunnyTweak/main/app-repo.json>)\n-# Note: TrollStore will display an encryption warning which you can ignore\n- **Jailbroken**\n - Tweak Debian Package - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n\nPlease keep in mind that this channel is not sideload-, jailbreak nor root support.\nPlease keep questions pertaining to those topics elsewhere."
 enable_experiments = "Head to Settings > Bunny > Activate Discord Experiments"
 account_switcher = "Enable Discord Experiments and look for the 'Mobile Account Switcher' experiment and set it to Treatment 1. Hold your profile in the navigation tab to manage your accounts."
 disable_nmu = "The only way to disable NMU is to use Discord version <= 224 and change the experiment's treatment. Keep in mind that support is only provided for the latest stable release and your version may stop working at some point."
-fix_voice_activity = "The cause of this is unclear. To fix it, please disable 'Advanced Voice Activity' in voice settings."
-no_mod_settings = "Try visiting [this link](<https://raw.githubusercontent.com/pyoncord/detta-builds/main/bunny.js>) and see if it loads bunch of texts. If it doesn't, try using a VPN and/or disable any sort of ad blocker and try again.\n-#These troubleshooting steps may still resolve your issue even if the link loads a bunch of text."
+fix_voice_activity = "The cause of this is unclear. To fix it, please disable `Advanced Voice Activity` in voice settings."
```

Comment:
feels redundant to remove the subtext but introduce a codeblock, should be either no markdown or keeping the markdown

Reply:
alright, adding codeblock on it.